src/vmware/HISTORY.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 4.0.0 (2021-12)
```

Comment:
Why is the date not the current month, but the next month?

Reply:
It should be this month. This isn't the final PR though.